### DISCUSSION TITLE
Area-Card: Alert-Class "Opening"

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#20988] [Dashboard] [answered]
can someone maybe explain, to what devices / entities the alert-class "opening" is linked?

I can select this class on the area card for our living room, where we do have a shutter / cover installed... but since "cover" entities are not binary, it seems not to be related with this device - also, because I have never seen any alarm information being shown.

In addition to that, I have setup another cover device for testing purposes, and can't add the "opening" alarm class to any other card (not even the area, the test device is assigned to)

I would just love to know, what's behind this "opening" alert :)

Thanks!

**Comment by karwosts** [comment, marked as answer]
All the area card alert classes are binary_sensors. If your menu has an option for `Opening`, then you have a binary sensor with `device_class: opening` assigned to that area. 

All the device classes are listed here. 
https://www.home-assistant.io/integrations/binary_sensor/

**Reply by ChristophCaina** [reply]
thanks!
Unbeleivable... it was a binary_sensor from a Lightbulb - that comes as "open" by default...
WTF...
